src/common/commonutils/PackageUtils.c

```diff
@@ -91,61 +91,104 @@ int CheckPackageInstalled(const char* packageName, void* log)
 
     if (0 == status)
     {
-        OsConfigLogInfo(log, "CheckPackageInstalled: '%s' is installed", packageName);
+        OsConfigLogInfo(log, "IsPackageInstalled: package '%s' is installed", packageName);
     }
     else
     {
-        OsConfigLogInfo(log, "CheckPackageInstalled: '%s' is not found", packageName);
+        OsConfigLogInfo(log, "IsPackageInstalled: package '%s' is not found", packageName);
     }
 
     return status;
 }
 
-int InstallPackage(const char* packageName, void* log)
+int CheckPackageInstalled(const char* packageName, char** reason, void* log)
```

Comment:
Should use `IsPackageInstalled` similar to your other call?

Reply:
Not the same thing.